# Czech translation for apport
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-12-07 16:23+0100\n"
"PO-Revision-Date: 2015-04-27 21:46+0000\n"
"Last-Translator: František Zatloukal <Zatloukal.Frantisek@gmail.com>\n"
"Language-Team: Czech <cs@li.org>\n"
"Language: cs\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2022-12-07 14:36+0000\n"
"X-Generator: Launchpad (build 41e7553f8097ce3a683f90e835708cbe7bf0006c)\n"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "Shromáždit informace o systému"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr ""
"Pro shromáždění informací o systému, potřebných k nahlášení tohoto problému, "
"je vyžadováno ověření"

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr "Zpráva o problému se systémem"

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid "Please enter your password to access problem reports of system programs"
msgstr ""
"Zadejte prosím své heslo, aby mohl být problém systémového programu oznámen."

#: ../apport/ui.py:165
msgid "This package does not seem to be installed correctly"
msgstr "Tento balíček zřejmě není nainstalován správně"

#: ../apport/ui.py:175
#, python-format
msgid ""
"This does not seem to be an official %s package. Please retry after updating "
"the indexes of available packages, if that does not work then remove related "
"third party packages and try again."
msgstr ""

#: ../apport/ui.py:205
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"Máte nainstalovány zastaralé verze některých balíčků. Prosím aktualizujte "
"následující balíčky a zkontrolujte, zda se problém vyskytuje i nadále.\n"
"\n"
"%s"

#: ../apport/ui.py:359
msgid "unknown program"
msgstr "neznámý program"

#: ../apport/ui.py:362
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "Omlouváme se, program \"%s\" byl neočekávaně uzavřen"

#: ../apport/ui.py:365 ../apport/ui.py:1899
#, python-format
msgid "Problem in %s"
msgstr "Problém v %s"

#: ../apport/ui.py:370
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"Váš počítač nemá dostatek volné paměti pro automatickou analýzu problému a "
"odeslání hlášení vývojářům."

#: ../apport/ui.py:430 ../apport/ui.py:448 ../apport/ui.py:621
#: ../apport/ui.py:630 ../apport/ui.py:867 ../apport/ui.py:1677
#: ../apport/ui.py:1860 ../apport/ui.py:1865
msgid "Invalid problem report"
msgstr "Neplatné hlášení o problému"

#: ../apport/ui.py:431
msgid "You are not allowed to access this problem report."
msgstr "Nemáte oprávnění k přístupu k tomuto hlášení problému."

#: ../apport/ui.py:439
msgid "Error"
msgstr "Chyba"

#: ../apport/ui.py:441
msgid "There is not enough disk space available to process this report."
msgstr "Pro zpracování tohoto hlášení není na disku dost místa."

#: ../apport/ui.py:479
msgid "No PID specified"
msgstr ""

#: ../apport/ui.py:481
msgid "You need to specify a PID. See --help for more information."
msgstr ""

#: ../apport/ui.py:492 ../apport/ui.py:597
msgid "Invalid PID"
msgstr "Neplatný PID"

#: ../apport/ui.py:493
msgid "The specified process ID does not exist."
msgstr "Proces s daným identifikátorem neexistuje."

#: ../apport/ui.py:498
msgid "Not your PID"
msgstr "Není vaše PID"

#: ../apport/ui.py:499
msgid "The specified process ID does not belong to you."
msgstr "Proces s daným identifikátorem vám nepatří."

#: ../apport/ui.py:554
msgid "No package specified"
msgstr "Nebyl specifikován žádný balík"

#: ../apport/ui.py:556
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""
"Musíte určit, o který balík nebo PID se jedná. Pro více informací použijte "
"přepínač --help."

#: ../apport/ui.py:584
msgid "Permission denied"
msgstr "Přístup odepřen"

#: ../apport/ui.py:586
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"Nejste vlastníkem vybraného procesu. Prosím, spusťte tuto aplikaci jako "
"vlastník onoho procesu nebo jako root."

#: ../apport/ui.py:599
msgid "The specified process ID does not belong to a program."
msgstr "Zadané ID procesu neodpovídá žádnému programu."

#: ../apport/ui.py:623
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "Skript %s neurčuje problematický balík"

#: ../apport/ui.py:631
#, python-format
msgid "Package %s does not exist"
msgstr "Balík %s neexistuje"

#: ../apport/ui.py:660 ../apport/ui.py:872 ../apport/ui.py:905
#: ../apport/ui.py:915
msgid "Cannot create report"
msgstr "Nelze vytvořit hlášení"

#: ../apport/ui.py:675 ../apport/ui.py:734 ../apport/ui.py:752
msgid "Updating problem report"
msgstr "Probíhá aktualizace chybového hlášení"

#: ../apport/ui.py:677
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"Tento problém jste buď nenahlásili a ani nejste přihlášeni k jeho odběru, "
"nebo je toto hlášení problém duplicitní nebo již uzavřené.\n"
"\n"
"Vytvořte prosím nové hlášení pomocí \"apport-bug\"."

#: ../apport/ui.py:689
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"Nejste ten, kdo nahlásil tento problém. Je mnohem jednodušší označit chybu "
"jako duplikát jiné, než přesouvat komentáře a přílohy k nově hlášené chybě.\n"
"\n"
"V návaznosti doporučujeme vyplnit nové hlášení o chybě pomocí \"apport-bug\" "
"a u stávajícího hlášení o chybě přidat poznámku o Vašem novém hlášení o "
"chybě.\n"
"\n"
"Opravdu chcete pokračovat?"

#: ../apport/ui.py:735 ../apport/ui.py:753
msgid "No additional information collected."
msgstr "Nebyly shromážděny žádné další informace."

#: ../apport/ui.py:816
msgid "What kind of problem do you want to report?"
msgstr "Jaký druh problému si přejete nahlásit?"

#: ../apport/ui.py:837
msgid "Unknown symptom"
msgstr "Neznámý příznak"

#: ../apport/ui.py:838
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "Příznak \"%s\" je neznámý."

#: ../apport/ui.py:874
msgid ""
"The window option cannot be used on Wayland.\n"
"\n"
"Please find the window's process ID and then run 'ubuntu-bug <process ID>'.\n"
"\n"
"The process ID can be found by running the System Monitor application. In "
"the Processes tab, scroll until you find the correct application. The "
"process ID is the number listed in the ID column."
msgstr ""
"Možnost okna nelze použít na Wayland.\n"
"\n"
"Najděte ID procesu okna a poté spusťte 'ubuntu-bug <process ID>'.\n"
"\n"
"ID procesu lze zjistit spuštěním aplikace Sledování systému. Na kartě "
"Procesy posouvejte, dokud nenajdete správnou aplikaci. ID procesu je číslo "
"uvedené ve sloupci ID."

#: ../apport/ui.py:890
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr "Po uzavření této zprávy klikněte na okno s nahlášením problému."

#: ../apport/ui.py:907 ../apport/ui.py:916
msgid "xprop failed to determine process ID of the window"
msgstr "xprop selhal při rozeznávávání ID procesu okna"

#: ../apport/ui.py:931
#, python-format
msgid "%(prog)s <report number>"
msgstr "%(prog)s <číslo hlášení>"

#: ../apport/ui.py:932
msgid "Specify package name."
msgstr "Upřesněte název balíku."

#: ../apport/ui.py:938 ../apport/ui.py:1070
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr "Přidejte do hlášení další tagy - může jich být více."

#: ../apport/ui.py:976
#, python-format
msgid ""
"%(prog)s [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""

#: ../apport/ui.py:987
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"Spustit režim popisování chyby. Vyžaduje přepínač --package a volitelně --"
"pid, nebo pouze --pid. Pokud není zadán ani jeden, zobrazí se seznam známých "
"příznaků. (Použito, pokud je zadán jediný argument.)"

#: ../apport/ui.py:996
msgid "Click a window as a target for filing a problem report."
msgstr "Kliknutím vyberte okno pro vytvoření hlášení o chybě."

#: ../apport/ui.py:1005
msgid "Start in bug updating mode. Can take an optional --package."
msgstr "Spustit režim aktualizace chyb.  Může mít volbu --packages"

#: ../apport/ui.py:1014
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""
"Podat hlášení o chybě o příznaku. (Použito, pokud je zadán název příznaku "
"jako jediný argument.)"

#: ../apport/ui.py:1023
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"Určit název balíku v režimu --file-bug. Toto je volitelné, pokud je určeno --"
"pid. (Použito, pokud je zadán název balíku jako jediný argument.)"

#: ../apport/ui.py:1034
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""
"Upřesněte běžící program v režimu --file-bug mode. S tímto upřesněním bude "
"hlášení obsahovat více informací. (Je-li číslo procesu uvedeno jako jediný "
"argument.)"

#: ../apport/ui.py:1042
msgid "The provided pid is a hanging application."
msgstr "Poskytnuté pid je zamrzlá aplikace."

#: ../apport/ui.py:1050
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"Podat hlášení o havárii ze zadaného souboru .apport nebo .crash místo "
"některých nevyřízených z %s. (Použito, pokud je zadán soubor jako jediný "
"argument.)"

#: ../apport/ui.py:1060
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""
"Uložte hlášení do souboru namísto přímého oznámení. Tento soubor může být "
"nahlášen později nebo z jiného počítače."

#: ../apport/ui.py:1078
msgid "Print the Apport version number."
msgstr "Vypsat číslo verze Apport."

#: ../apport/ui.py:1240
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr "Tímto spustíte apport-retrace v terminálovém okně."

#: ../apport/ui.py:1244
msgid "Run gdb session"
msgstr "Spustit gdb"

#: ../apport/ui.py:1245
msgid "Run gdb session without downloading debug symbols"
msgstr "Spustit gdb bez stahování debugovacích symbolů"

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:1247
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr "Aktualizovat %s pomocí úplného symbolického trasování zásobníku"

#: ../apport/ui.py:1309
msgid "Can't remember send report status settings"
msgstr "Nenalezeno nastavení stavu odeslání hlášení"

#: ../apport/ui.py:1313
msgid ""
"Saving crash reporting state failed. Can't set auto or never reporting mode."
msgstr ""
"Ukládání stavu hlášení o chybě selhalo. Nelze nastavit mód hlášení na "
"automaticky nebo nikdy."

#: ../apport/ui.py:1396 ../apport/ui.py:1409
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr "Toto hlášení o problému se týká programu, který již není nainstalován."

#: ../apport/ui.py:1433
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr "Problém byl zaznamenán v programu %s, který byl od pádu změněn."

#: ../apport/ui.py:1494 ../apport/ui.py:1618 ../apport/ui.py:1869
msgid "This problem report is damaged and cannot be processed."
msgstr "Toto hlášení o problému je poškozené a nemůže být zpracováno."

#: ../apport/ui.py:1503
msgid "This report is about a package that is not installed."
msgstr "Toto hlášení je o balíku, který není nainstalován."

#: ../apport/ui.py:1511
msgid "An error occurred while attempting to process this problem report:"
msgstr "Při zpracování chybového hlášení došlo k chybě:"

#: ../apport/ui.py:1528
msgid ""
"You have two versions of this application installed, which one do you want "
"to report a bug against?"
msgstr ""

#: ../apport/ui.py:1533
#, python-format
msgid "%s snap"
msgstr ""

#: ../apport/ui.py:1534
#, python-format
msgid "%s deb package"
msgstr ""

#: ../apport/ui.py:1575
#, python-format
msgid "%s is provided by a snap published by %s. Contact them via %s for help."
msgstr ""

#: ../apport/ui.py:1580
#, python-format
msgid ""
"%s is provided by a snap published by %s. No contact address has been "
"provided; visit the forum at https://forum.snapcraft.io/ for help."
msgstr ""

#: ../apport/ui.py:1679
msgid "Could not determine the package or source package name."
msgstr "Nelze určit název balíku nebo zdrojového balíku."

#: ../apport/ui.py:1704
msgid "Unable to start web browser"
msgstr "Nelze spustit webový prohlížeč"

#: ../apport/ui.py:1705
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "Nelze spustit webový prohlížeč a otevřít %s."

#: ../apport/ui.py:1815
msgid "Network problem"
msgstr "Problém se sítí"

#: ../apport/ui.py:1819
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr ""
"Nelze se připojit k databází havárií, prosím zkontrolujte své internetové "
"připojení."

#: ../apport/ui.py:1851
msgid "Memory exhaustion"
msgstr "Vyčerpání paměti"

#: ../apport/ui.py:1853
msgid "Your system does not have enough memory to process this crash report."
msgstr ""
"Váš systém nemá dostatek paměti pro zpracování tohoto hlášení o havárii."

#: ../apport/ui.py:1904
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"Problém nemůže být nahlášen:\n"
"\n"
"%s"

#: ../apport/ui.py:1960 ../apport/ui.py:1972
msgid "Problem already known"
msgstr "Problém je již znám"

#: ../apport/ui.py:1962
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"Tento problém byl již nahlášen v hlášení zobrazeném ve vašem webovém "
"prohlížeči. Prosím zkontrolujte, zda můžete přidat další informace, které by "
"mohly být vývojářům nápomocné."

#: ../apport/ui.py:1974
msgid "This problem was already reported to developers. Thank you!"
msgstr "Tato chyba byla vývojářům již ohlášena. Děkujeme!"

#: ../bin/apport-cli.py:84
msgid "Press any key to continue..."
msgstr "Pro pokračování stiskněte libovolnou klávesu..."

#: ../bin/apport-cli.py:91
msgid "What would you like to do? Your options are:"
msgstr "Co chcete udělat? Máte tyto možnosti:"

#: ../bin/apport-cli.py:104
#, python-format
msgid "Please choose (%s):"
msgstr "Prosím vyberte (%s):"

#: ../bin/apport-cli.py:172
#, python-format
msgid "(%i bytes)"
msgstr "(%i bajtů)"

#: ../bin/apport-cli.py:174 ../gtk/apport-gtk.py:169 ../kde/apport-kde.py:406
msgid "(binary data)"
msgstr "(binární data)"

#: ../bin/apport-cli.py:209 ../gtk/apport-gtk.py:208 ../gtk/apport-gtk.ui.h:6
#: ../kde/apport-kde.py:199
msgid "Send problem report to the developers?"
msgstr "Odeslat hlášení o problému vývojářům?"

#: ../bin/apport-cli.py:211
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"Po odeslání hlášení o problému vyplňte prosím formulář v nově\n"
"otevřeném okně prohlížeče."

#: ../bin/apport-cli.py:218
#, python-format
msgid "&Send report (%s)"
msgstr "&Odeslat hlášení (%s)"

#: ../bin/apport-cli.py:223
msgid "&Examine locally"
msgstr "&Prozkoumat místně"

#: ../bin/apport-cli.py:227
msgid "&View report"
msgstr "Zobrazi&t hlášení"

#: ../bin/apport-cli.py:230
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr ""
"&Ponechat soubor s hlášením pro pozdější odeslání nebo kopírování někam jinam"

#: ../bin/apport-cli.py:235
msgid "Cancel and &ignore future crashes of this program version"
msgstr "Zrušit a &ignorovat budoucí havárie této verze programu"

#: ../bin/apport-cli.py:238 ../bin/apport-cli.py:332 ../bin/apport-cli.py:366
#: ../bin/apport-cli.py:389
msgid "&Cancel"
msgstr "&Zrušit"

#: ../bin/apport-cli.py:273
msgid "Problem report file:"
msgstr "Soubor s hlášením o problému:"

#: ../bin/apport-cli.py:279 ../bin/apport-cli.py:284
msgid "&Confirm"
msgstr "&Potvrdit"

#: ../bin/apport-cli.py:283
#, python-format
msgid "Error: %s"
msgstr "Chyba: %s"

#: ../bin/apport-cli.py:289 ../kde/apport-kde.py:462
msgid "Collecting problem information"
msgstr "Shromažďování informací o problému"

#: ../bin/apport-cli.py:291
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"Sesbírané informace mohou být odeslány vývojářům za účelem zlepšení\n"
"aplikace. To může trvat i několik minut."

#: ../bin/apport-cli.py:307 ../gtk/apport-gtk.ui.h:17 ../kde/apport-kde.py:511
msgid "Uploading problem information"
msgstr "Odesílání informací o problému"

#: ../bin/apport-cli.py:309
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"Sesbírané informace se odesílají do systému pro sledování chyb.\n"
"To může trvat i několik minut."

#: ../bin/apport-cli.py:365
msgid "&Done"
msgstr "&Hotovo"

#: ../bin/apport-cli.py:371
msgid "none"
msgstr "nic"

#: ../bin/apport-cli.py:373
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "Vybráno: %s. Více možností:"

#: ../bin/apport-cli.py:390
msgid "Choices:"
msgstr "Volby"

#: ../bin/apport-cli.py:404
msgid "Path to file (Enter to cancel):"
msgstr "Cesta k souboru (Enter pro zrušení):"

#: ../bin/apport-cli.py:410
msgid "File does not exist."
msgstr "Soubor neexistuje."

#: ../bin/apport-cli.py:412
msgid "This is a directory."
msgstr "Toto je adresář."

#: ../bin/apport-cli.py:418
msgid "To continue, you must visit the following URL:"
msgstr "Pro pokračování musíte navštívit následující adresu:"

#: ../bin/apport-cli.py:421
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"Nyní můžete spustit prohlížeč nebo zkopírovat tuto adresu do prohlížeče nebo "
"jiného počítače."

#: ../bin/apport-cli.py:427
msgid "Launch a browser now"
msgstr "Spustit prohlížeč nyní"

#: ../bin/apport-cli.py:443
msgid "No pending crash reports. Try --help for more information."
msgstr ""
"Žádná nevyřízená hlášení o haváriích. Pro více informací použijte přepínač --"
"help."

#: ../bin/apport-retrace.py:44
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr ""
"Nepřidávejte do hlášení další sledování, přesměrujte je na standardní výstup."

#: ../bin/apport-retrace.py:53
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""
"Spusťte interaktivní gdb úlohu s hlášením core dump (-případně ignorujte; "
"nepřepisujte report)"

#: ../bin/apport-retrace.py:62
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""
"Zapsat upravené hlášení do daného souboru místo změny původního hlášení"

#: ../bin/apport-retrace.py:72
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr ""
"Odstraňte core dump z hlášení potom, co bude výstupní hlášení znovu "
"vygenerováno"

#: ../bin/apport-retrace.py:80
msgid "Override report's CoreFile"
msgstr "Potlačit zprávy CoreFile"

#: ../bin/apport-retrace.py:86
msgid "Override report's ExecutablePath"
msgstr "Potlačit zprávy ExecutablePath"

#: ../bin/apport-retrace.py:92
msgid "Override report's ProcMaps"
msgstr "Potlačit zprávy ProcMaps"

#: ../bin/apport-retrace.py:98
msgid "Rebuild report's Package information"
msgstr "Obnovit zprávy o informacích o balíčcích"

#: ../bin/apport-retrace.py:105
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""
"Vytvořit dočasný sandbox a stáhnout/instalovat tam potřebné balíčky a "
"symboly pro ladění; bez této volby bude předpokládat, že potřebné balíčky a "
"symboly pro ladění jsou již v systému instalovány. Argument odkazuje na "
"kořenový adresář s konfigurací balíkovacího systému; pokud uvedete "
"\"system\", tak budou použity systémové konfigurační soubory, ale pak bude "
"možné retrasovat havárie pouze u právě spuštěné verze."

#: ../bin/apport-retrace.py:119
msgid ""
"Build another temporary sandbox for installing gdb and its dependencies "
"using the same release as the report rather than whatever version of gdb you "
"have installed."
msgstr ""
"Vytvořit další dočasný sandbox pro instalaci gdb a závislostí za použití "
"stejného vydání jako v hlášení namísto verze gdb, která je nainstalována."

#: ../bin/apport-retrace.py:129
msgid "Report download/install progress when installing packages into sandbox"
msgstr "Hlásit postup stahování/instalace při instalaci balíčků do sandboxu"

#: ../bin/apport-retrace.py:136
msgid "Prepend timestamps to log messages, for batch operation"
msgstr "Pro dávkové zpracování vložit časové razítko před zpávu do logu"

#: ../bin/apport-retrace.py:142
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr "Vytvořit a použít repozitáře třetích stran pocházející z hlášení"

#: ../bin/apport-retrace.py:150
msgid "Cache directory for packages downloaded in the sandbox"
msgstr "Kešovací adresář pro stažené balíčky do sandboxu"

#: ../bin/apport-retrace.py:156
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""
"Adresář pro rozbalené soubory. V budoucnu by tam měly být k dispozici "
"všechny stažené a rozbalené balíky."

#: ../bin/apport-retrace.py:167 ../bin/apport-valgrind.py:97
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr "Nainstalovat do sandboxu extra balík (může být vybráno vícekrát)"

#: ../bin/apport-retrace.py:174
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""
"Cesta k souboru s pověřením k databázi havárií. Při definování ID havárie je "
"použit k nahrání retrasovaných trasování zásobníku (pouze pokud není použit "
"parametr -g, -o nebo -s)"

#: ../bin/apport-retrace.py:184
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""
"Zobrazit retasované trasování zásobníku a poptat na potvrzení před jejich "
"odesláním do databáze havárií."

#: ../bin/apport-retrace.py:192
msgid "Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr "Cesta k duplikátu databáze sqlite (výchozí: žádná kontrola duplicit)"

#: ../bin/apport-retrace.py:200
msgid "Do not add StacktraceSource to the report."
msgstr "Nepřidávejte do hlášení StacktraceSource."

#: ../bin/apport-retrace.py:213
msgid "You cannot use -C without -S. Stopping."
msgstr "Nelze užít -C bez -S. Zastavuji proces."

#. translators: don't translate y/n,
#. apport currently only checks for "y"
#: ../bin/apport-retrace.py:243
msgid "OK to send these as attachments? [y/n]"
msgstr "Souhlasíte s odesláním těchto jako příloh? [y/n]"

#: ../bin/apport-unpack.py:29
#, python-format
msgid "%(prog)s <report> <target directory>"
msgstr "%(prog)s <hlášení> <cílový adresář>"

#: ../bin/apport-unpack.py:31
msgid "Report file to unpack"
msgstr "Soubor s hlášením k rozbalení"

#: ../bin/apport-unpack.py:33
msgid "directory to unpack report to"
msgstr "adresář, do kterého se má hlášení rozbalit"

#: ../bin/apport-unpack.py:59
msgid "Destination directory exists and is not empty."
msgstr "Cílový adresář existuje a není prázdný."

#: ../bin/apport-valgrind.py:37
msgid "See man page for details."
msgstr "Více informací je na manuálové stránce"

#: ../bin/apport-valgrind.py:46
msgid "specify the log file name produced by valgrind"
msgstr "specifikujte jméno log souboru valgrindu"

#: ../bin/apport-valgrind.py:52
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""
"použít dříve vytvořený adresář sandbox (SDIR) nebo, pokud neexistuje, jej "
"vytvořit"

#: ../bin/apport-valgrind.py:60
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""
"nevytvářet nebo znovu nepoužívat adresář sandbox pro dodatečné debugovací "
"symboly, ale pouze pro instalované debugovací symboly."

#: ../bin/apport-valgrind.py:69
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""
"použít dříve vytvořený cache adresář (CDIR) nebo, pokud neexistuje, jej "
"vytvořit"

#: ../bin/apport-valgrind.py:78
msgid "report download/install progress when installing packages into sandbox"
msgstr "hlásit postup stahování/instalace během instalování balíku do sandboxu"

#: ../bin/apport-valgrind.py:86
msgid ""
"the executable that is run under valgrind's memcheck tool for memory leak "
"detection"
msgstr ""
"spustitelný soubor, který spouští nástroj memcheck programu valgrind k "
"detekci úniků paměti"

#: ../bin/apport-valgrind.py:129
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr "Chyba: %soubor není spustitelný. Ukončuji proces."

#: ../data/apportcheckresume.py:68
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr ""
"Toto se stalo v průběhu minulého uspání do paměti a zabránilo systému v "
"probuzení."

#: ../data/apportcheckresume.py:73
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr ""
"Toto se stalo v průběhu minulého uspání na disk a zabránilo systému v "
"probuzení."

#: ../data/apportcheckresume.py:81
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""
"Proces probouzení byl porušen téměř u konce a vypadá tedy jako řádně "
"ukončený."

#: ../data/kernel_oops.py:30
msgid "Your system might become unstable now and might need to be restarted."
msgstr ""
"Váš systém může být nyní nestabilní a bude jej možná nutné restartovat."

#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde-mime.desktop.in.h:1
#: ../kde/apport-kde.desktop.in.h:1
msgid "Report a problem..."
msgstr "Nahlásit problém…"

#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde-mime.desktop.in.h:2
#: ../kde/apport-kde.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "Oznámit problém vývojářům"

#: ../gtk/apport-gtk.py:185
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "Omlouváme se, aplikace %s byla neočekávaně ukončena."

#: ../gtk/apport-gtk.py:188
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "Omlouváme se, %s byl neočekávaně ukončen."

#: ../gtk/apport-gtk.py:193 ../kde/apport-kde.py:214 ../kde/apport-kde.py:266
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "Omlouváme se, %s zaznamenal vnitřní chybu."

#: ../gtk/apport-gtk.py:216 ../gtk/apport-gtk.ui.h:14 ../kde/apport-kde.py:207
msgid "Send"
msgstr "Odeslat"

#: ../gtk/apport-gtk.py:234 ../gtk/apport-gtk.py:666 ../gtk/apport-gtk.ui.h:11
#: ../kde/apport-kde.py:330
msgid "Show Details"
msgstr "Zobrazit podrobnosti"

#: ../gtk/apport-gtk.py:247 ../kde/apport-kde.py:258 ../kde/apport-kde.py:276
msgid "Continue"
msgstr "Pokračovat"

#: ../gtk/apport-gtk.py:271
#, python-format
msgid "The application %s has stopped responding."
msgstr "Aplikace %s neodpovídá."

#: ../gtk/apport-gtk.py:275
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "Program %s neodpovídá."

#: ../gtk/apport-gtk.py:292 ../kde/apport-kde.py:222
#, python-format
msgid "Package: %s"
msgstr "Balíček: %s"

#: ../gtk/apport-gtk.py:298 ../kde/apport-kde.py:229
msgid "Sorry, a problem occurred while installing software."
msgstr "Omlouváme se, při instalaci softwaru nastala chyba."

#: ../gtk/apport-gtk.py:310 ../gtk/apport-gtk.py:334 ../kde/apport-kde.py:238
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "V aplikaci %s došlo k chybě."

#: ../gtk/apport-gtk.py:316 ../kde/apport-kde.py:245
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "Aplikace %s byla neočekávaně ukončena."

#: ../gtk/apport-gtk.py:347 ../gtk/apport-gtk.ui.h:7 ../kde/apport-kde.py:271
msgid "If you notice further problems, try restarting the computer."
msgstr "Pokud problémy neustanou, zkuste restartovat počítač."

#: ../gtk/apport-gtk.py:352 ../kde/apport-kde.py:278
msgid "Ignore future problems of this type"
msgstr "Ignorovat budoucí problémy tohoto typu"

#: ../gtk/apport-gtk.py:670 ../kde/apport-kde.py:327
msgid "Hide Details"
msgstr "Skrýt podrobnosti"

#: ../gtk/apport-gtk.ui.h:1 ../kde/apport-kde.py:552 ../kde/apport-kde.py:595
msgid "Apport"
msgstr "Apport"

#: ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr "Zrušit"

#: ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr "OK"

#: ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr "Hlášení o chybě"

#: ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr "<big><b>Omlouváme se, nastala vnitřní chyba.</b></big>"

#: ../gtk/apport-gtk.ui.h:8
msgid "Remember this in future"
msgstr "Pamatovat si toto v budoucnu"

#: ../gtk/apport-gtk.ui.h:9
msgid "Ignore future problems of this program version"
msgstr "Ignorovat budoucí problémy této verze programu"

#: ../gtk/apport-gtk.ui.h:10
msgid "Relaunch this application"
msgstr "Znovu spustit tuto aplikaci"

#: ../gtk/apport-gtk.ui.h:12
msgid "_Examine locally"
msgstr "_Prozkoumat místně"

#: ../gtk/apport-gtk.ui.h:13
msgid "Don't send"
msgstr "Neodesílat"

#: ../gtk/apport-gtk.ui.h:15
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>Shromažďování informací o problému</b></big>"

#: ../gtk/apport-gtk.ui.h:16
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"Shromažďují se informace, které mohou pomoci vývojářům opravit vámi "
"nahlášený problém."

#: ../gtk/apport-gtk.ui.h:18
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>Odesílání informací o problému</b></big>"

#: ../gtk/apport-gtk.ui.h:19 ../kde/apport-kde.py:513
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"Sesbírané informace se odesílají do systému pro sledování chyb. To může "
"trvat i několik minut."

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Soubor apport crash"

#: ../kde/apport-kde.py:261
msgid "Leave Closed"
msgstr "Ponechat uzavřené"

#: ../kde/apport-kde.py:262 ../kde/apport-kde.py:432
msgid "Relaunch"
msgstr "Restartovat"

#: ../kde/apport-kde.py:355
msgid "Username:"
msgstr "Uživatelské jméno:"

#: ../kde/apport-kde.py:356
msgid "Password:"
msgstr "Heslo:"

#: ../kde/apport-kde.py:461
msgid "Collecting Problem Information"
msgstr "Shromažďování informací o problému"

#: ../kde/apport-kde.py:464
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""
"Shromážděné informace mohou být zaslány vývojářům za účelem případného "
"vylepšení aplikace. Odesílání může trvat několik minut."

#: ../kde/apport-kde.py:510
msgid "Uploading Problem Information"
msgstr "Odesílání informací o problému"
